### DISCUSSION TITLE
Read status on start didn't work

### DISCUSSION THREAD

**Discussion by 19grad** [Supergiovane/node-red-contrib-knx-ultimate#409] [General]
Hello everyone,

I try to configure a persistent Group Adresse

With:
Read status on start: Read the last value saved to file prior to disconnection

![2024-12-14 15_33_11-Node-RED _ Anwesend Status](https://github.com/user-attachments/assets/54da1dcf-d901-4c81-930e-55514c4197de)

Status true before redeploy
![2024-12-14 15_27_25-Node-RED _ Anwesend Status](https://github.com/user-attachments/assets/da4cbd9d-c9f2-4426-ba6e-b77234c47fb3)

After e redeploy of nodeRed the the status of the GA is gone
![2024-12-14 15_28_44-Node-RED _ Anwesend Status](https://github.com/user-attachments/assets/1a8b0bb0-097b-4dbf-b1ce-830b3221d04a)

After a read request from the bus
The responce to the Bus is false
![2024-12-14 15_29_26-Node-RED _ Anwesend Status](https://github.com/user-attachments/assets/6e7a24e7-97da-4992-9cb4-e36cbbf47ac4)

I would expected that the true Status of the GA is persistent over the redeploy

**Knx-Ultimate Version**
3.3.8
Node-Red running in a docker container

Thanks in advance

**Comment by Supergiovane** [comment]
Hi
Everytime you enter the node configuration page, the value is deleted. This is an intended behavior.

**Reply by 19grad** [reply]
Hello 

I didn't entered the node configuration page
I only press the deploy button



**Test 1:**
Read status on start: Read from the KNX Bus ...
No future opening of the configuration page
Pressing the NodeRed deploy button

Set the GA too true, and check
GroupValueWrite from ETS: TRUE
GroupValueRead from ETS: GroupeValueResponce from nodeRed tunnel: TRUE

Pressing the NodeRed deploy button

ETS GA Monitor; GroupValueRead from the nodeRed Tunnel
ETS GA Monitor; GroupValueResponce FALSE from the nodeRed Tunnel

>> the configuration worked as i expected
knx ultimate send the GroupValueRead  to the bus, and answered to himself
of course the value is lost


**Test 2:**
Read status on start: Read the last value saved to file prior to disconnection
No future opening of the configuration page
Pressing the NodeRed deploy button

Set the GA too true, and check
GroupValueWrite from ETS: TRUE
GroupValueRead from ETS: GroupeValueResponce from nodeRed tunnel: TRUE

Pressing the NodeRed deploy button

GroupValueRead from ETS: GroupeValueResponce from nodeRed tunnel: FALSE

>> in this case i would expect that knx ultimate would have saved the last TRUE value
instead he responced with the default value


did i configure something wrong?
Or did i misunderstand the function?
